cwms-data-api/src/test/java/cwms/cda/data/dao/location/kind/OutletDaoUncontrolledIT.java

```diff
@@ -0,0 +1,209 @@
+    private static final CwmsId SPILLWAY_LOW_FLOW_RATING_GROUP = new CwmsId.Builder()
+            .withName("Rating-" + PROJECT_1_ID.getName() + "-LowFlow").withOfficeId(PROJECT_1_ID.getOfficeId()).build();
+    private static final CwmsId SPILLWAY_LOW_FLOW_RATING_GROUP_MODIFIED = new CwmsId.Builder()
+            .withName("Rating-" + PROJECT_1_ID.getName() + "-LowFlow-Modified")
+            .withOfficeId(PROJECT_1_ID.getOfficeId()).build();
+    private static final Location SPILLWAY_LOW_FLOW_1_LOC
+            = buildProjectStructureLocation(PROJECT_1_ID.getName() + "-LF1", OUTLET_KIND);
+    private static final Location SPILLWAY_LOW_FLOW_2_LOC
+            = buildProjectStructureLocation(PROJECT_1_ID.getName() + "-LF2", OUTLET_KIND);
+    private static final Location SPILLWAY_LOW_FLOW_3_LOC
+            = buildProjectStructureLocation(PROJECT_1_ID.getName() + "-LF3", OUTLET_KIND);
+    private static final Outlet SPILLWAY_LOW_FLOW_1_OUTLET = buildTestOutlet(SPILLWAY_LOW_FLOW_1_LOC,
+            PROJECT_LOC, SPILLWAY_LOW_FLOW_RATING_GROUP);
+    private static final Outlet SPILLWAY_LOW_FLOW_2_OUTLET = buildTestOutlet(SPILLWAY_LOW_FLOW_2_LOC,
+            PROJECT_LOC, SPILLWAY_LOW_FLOW_RATING_GROUP);
+
+    @BeforeAll
+    static void setup() throws Exception {
+        setupProject();
+        CwmsDatabaseContainer<?> databaseLink = CwmsDataApiSetupCallback.getDatabaseLink();
+        databaseLink.connection(c -> {
+            DSLContext context = getDslContext(c, OFFICE_ID);
+            OutletDao outletDao = new OutletDao(context);
+            try {
+                storeLocation(context, SPILLWAY_LOW_FLOW_1_LOC);
+                storeLocation(context, SPILLWAY_LOW_FLOW_2_LOC);
+                outletDao.storeOutlet(SPILLWAY_LOW_FLOW_1_OUTLET, false);
+                outletDao.storeOutlet(SPILLWAY_LOW_FLOW_2_OUTLET, false);
+            } catch (IOException e) {
+                throw new RuntimeException(e);
+            }
+        }, CwmsDataApiSetupCallback.getWebUser());
+    }
+
+    @AfterAll
+    static void cleanup() throws Exception {
+        CwmsDatabaseContainer<?> databaseLink = CwmsDataApiSetupCallback.getDatabaseLink();
+        databaseLink.connection(c -> {
+            DSLContext context = getDslContext(c, OFFICE_ID);
+            OutletDao outletDao = new OutletDao(context);
+            outletDao.deleteOutlet(SPILLWAY_LOW_FLOW_1_LOC.getOfficeId(), SPILLWAY_LOW_FLOW_1_LOC.getName(),
+                    DeleteRule.DELETE_ALL);
+            outletDao.deleteOutlet(SPILLWAY_LOW_FLOW_2_LOC.getOfficeId(), SPILLWAY_LOW_FLOW_2_LOC.getName(),
+                    DeleteRule.DELETE_ALL);
+            deleteLocation(context, SPILLWAY_LOW_FLOW_1_LOC.getOfficeId(), SPILLWAY_LOW_FLOW_1_LOC.getName());
+            deleteLocation(context, SPILLWAY_LOW_FLOW_2_LOC.getOfficeId(), SPILLWAY_LOW_FLOW_2_LOC.getName());
+        }, CwmsDataApiSetupCallback.getWebUser());
+        tearDownProject();
+    }
+
+    @Test
```

Comment:
To 198, we don't need, this is already tested

Reply:
Removed and replaced with proper test cases